bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Observation.scala

```diff
@@ -11,6 +11,7 @@ object Observation {
   val condition:Lens[Observation, Option[Condition]] = Lens.lensu((a, b) => a.copy(condition = b), _.condition)
   val target:Lens[Observation, Option[Target]] = Lens.lensu((a, b) => a.copy(target = b), _.target)
   val meta:Lens[Observation, Option[ObservationMeta]] = Lens.lensu((a, b) => a.copy(meta = b), _.meta)
+  val time:Lens[Observation, Option[TimeAmount]] = Lens.lensu((a, b) => a.copy(time = b), _.time)
```

Comment:
State of the art is a library by Julien Truffaut called [Monocle](https://github.com/julien-truffaut/Monocle). I would like to integrate it once we have upgraded to scalaz 7.1 (or 7.2, depending on when it happens).